packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.tsx

```diff
@@ -45,6 +48,14 @@ export class GuxFormFieldColor {
   @Prop()
   labelPosition: GuxFormFieldLabelPosition;
 
+  /**
+   * Field indicator mark which can show *, (optional) or blank
+   * Defaults to required. When set to required, the component will display * for required fields and blank for optional
+   * When set to optional, the component will display (optional) for optional and blank for required.
+   */
+  @Prop()
+  indicatorMark: GuxFormFieldIndicatorMark;
```

Comment:
I dont think this currently defaults to `required`. I think it would have to be `indicatorMark: GuxFormFieldIndicatorMark = 'required'`

Reply:
The default is set in the label component so I don't think it's needed here.

Reply:
Ahh okay makes sense, is there a reason for setting it within the functional component ? I guess it fine as there is a comment explaining that it is set to `required` by default.

Reply:
Didn't really have a preference but I updated it to take the default from the component instead.